core/src/main/res/values/themes.xml

```diff
@@ -17,7 +17,7 @@
     <item name="colorSecondaryVariant">?colorPrimary</item>
 
     <!--Material snackbar attributes-->
-    <item name="maxActionInlineWidth" tools:ignore="PrivateResource">@dimen/design_snackbar_action_inline_max_width</item>
+    <item name="maxActionInlineWidth">@dimen/snackbar_action_inline_max_width</item>
```

Comment:
It doesn't need to be a dimen either, you can put 128dp in here, I highly doubt we will ever need to reuse this

Reply:
@macgills done.